bin/test_sites

```diff
@@ -30,6 +30,7 @@ function redact_files {
     for file in $@; do
         sed -E -i \
             -e 's-While loading.*udmi/tests/-REDACTED/-' \
+            -e 's/at .*main\(.*\.java:[0-9]+\)/REDACTED/' \
```

Comment:
> What are you redacting, here? It looks like part of a stack trace?

See tests/sites/downgrade/devices/DWN-1/expected/exceptions.txt -- the stack trace is already redacted, this is just then adding in an obvious REDACTED marker... I was getting confused because the output was just removed before so just looking at that file was confuzel. Now, at least there's a big REDACTED marker to clarify.